### ISSUE TITLE
Choose Snapshot Attributes

### ISSUE THREAD

**Issue by moul** [atomone-hub/genesis#18]
Should we use the block at the end of the vote based on date [18010654](https://www.mintscan.io/cosmos/block/18010654)? Or should we consider an alternative? Can someone double-check if I have selected the correct block?

Once we confirm the correct block, I can help create a snapshot with all the metadata. We can process this snapshot later when we finalize the rules.

**Comment by giunatale** [comment]
I think the right block is either [18010660](https://www.mintscan.io/cosmos/block/18010660) (where the tallying is completed, you can see it because of the delay with the previous block is over 2 minutes instead of the usual 6 seconds) or block [18010659](https://www.mintscan.io/cosmos/block/18010659) which is where the tallying is triggered. The proposal was scheduled to end at 21:00:28-ish UTC which funnily enough concides with the time of block [18010658](https://www.mintscan.io/cosmos/block/18010658), so that's why it happens one block later.

But, tbh I am advocating for checking also other proposals and not just prop 848 (e.g. prop 69, 82)

**Comment by FocSon** [comment]
I am in favor of checking also props 69 and 82 as they follow the ideology of atomone.
I think that we can weigh the snapshot in function of the vote on a template like this one :

Veto 100%
No 90%
Abstain 50%
Yes 15% (voices against the idea that we share with atomone are important as they bring reflection)

Unbounding and liquid atom : 50% (as Jake said for the gnoland snapshot, liquid atom holders suffer enough from dilution to be punished by a 0% weight on this distribution. In the same idea they would be the one who can gold atomeone inflation. They represent a large amount of atom holders and I think that it is important to keep them with us in order to have the most decentralized distribution as possible)

Osmosis, neutron (and kujira ?) LP / lenders (mars, UX,...) : 50% (these people are the one who will be most likely to bring liquidity to atomone where atom stakers will most likely stake their atomeone tokens).

Have I forgot a holder category ?

**Comment by giunatale** [comment]
Hey @FocSon, this was strictly to deterine snapshot(s) block height.
The discussion around the distribution is being held in https://github.com/atomone-hub/genesis/issues/12

In general though what you are saying seems fairly reasonable. Please carry on the discussion there 🙏

**Comment by giunatale** [comment]
So for proposal 82, if we also want to snapshot the data for that one, it would be either block [12842181](https://www.mintscan.io/cosmos/block/12842181) - where the tally is triggered - or [12842182](https://www.mintscan.io/cosmos/block/12842182) - when the tally is finalized, following the same reasoning.

Just wanted to leave this here in case needed / helpful.

**Comment by albttx** [comment]
I can manage to setup a node with a snapshots at this block :)

**Comment by discoverdefiteam** [comment]
@giunatale hi! seems the snapshot should refer to when the tally was triggered, if it is true that vote decisions couldn't have been altered from the time the tally was triggered to the time it was finalized, therefore block [18010659](https://www.mintscan.io/cosmos/block/18010659)

This should also apply to [prop 82](https://www.mintscan.io/cosmos/proposals/82) if decided to be included in gauging AtomOne genesis distribution, meaning block [12842181](https://www.mintscan.io/cosmos/block/12842181). I want to provide my suggestion to this point specifically, will direct that comment to the proper issue thread.

**Comment by giunatale** [comment]
Yeah you can see also that votes in both blocks that are for the respective proposal fail because "proposal is inactive", therefore I agree that those blocks are sufficient.

**Comment by moul** [comment]
@albttx, yes please 👍

**Comment by albttx** [comment]
Some validators and addresses decided to do a [Weighted Vote](https://docs.cosmos.network/main/build/modules/gov#weighted-votes)

How should we count them ? Pro-rata of the no vote or slashed ?

![image](https://github.com/atomone-hub/genesis/assets/8089712/a65dccea-7543-4c3f-8486-da2e3251c659)

**Comment by moul** [comment]
The snapshot should reflect the reality with raw data, so we can decide this later.

My current opinion:
- Initially, let's involve only those who are pure/true NOers (NO + NoWithVeto).
- Then, we can let this NOers community decide if they wish to invite completely, partially, or not at all the weighted opinions.

**Comment by giunatale** [comment]
> * Initially, let's involve only those who are pure/true NOers (NO + NoWithVeto).

I think weighted vote are to be considered "pure" on the percentage that is assigned to NO/NWV. Don't really understand this assertion.

If they vote 30% no and 10% NWV, the stake counted is 30% and 10% respectively of the total. I don't see this as a technical challenge tbh
